src/electron/providers/storage/localFileSystem.ts

```diff
@@ -93,8 +93,18 @@ export default class LocalFileSystem implements IStorageProvider {
         });
     }
 
-    public listFiles(folderPath: string): Promise<string[]> {
-        return this.listItems(path.normalize(folderPath), (stats) => !stats.isDirectory());
+    public async listFiles(folderPath: string, ext?: string, recursive: boolean = false): Promise<string[]> {
```

Comment:
Same, any reason not do always have this as true?

Reply:
Wasn't sure. Was leaving it open for flexibility, but I'll remove that param